bee-message/bee-byte-cost-derive/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "bee-byte-cost-derive"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2021"
+description = "Internal crate that adds derive macros for computing the byte cost of outputs."
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = [ "iota", "tangle", "bee", "framework", "bytecost", "dust" ]
+homepage = "https://www.iota.org"
+publish = false
+
+[lib]
+proc-macro = true
+
+[dependencies]
+proc-macro-error = { version = "1.0.4", default-features = false, features = [ "syn-error" ] }
+proc-macro2 = { version = "1.0.30", default-features = false }
+quote = { version = "1.0.10", default-features = false }
+syn = { version = "1.0.80", default-features = false, features = [ "clone-impls", "derive", "parsing", "printing", "proc-macro" ] }
+
+[dev-dependencies]
+bee-byte-cost = { path = "../bee-byte-cost", version = "0.1.0", default-features = false }
+packable = { version = "0.1.0", default-features = false }
```

Comment:
I think `packable` now goes on the second group

Reply:
Indeed